ql/src/test/results/clientpositive/perf/tez/cbo_query14.q.out

```diff
@@ -251,7 +251,7 @@ HiveSortLimit(sort0=[$0], sort1=[$1], sort2=[$2], sort3=[$3], dir0=[ASC], dir1=[
                             HiveProject($f0=[$0], $f1=[$1], $f2=[$2])
                               HiveFilter(condition=[=($3, 3)])
                                 HiveAggregate(group=[{0, 1, 2}], agg#0=[count($3)])
-                                  HiveProject(i_brand_id=[$0], i_class_id=[$1], i_category_id=[$2], $f3=[$3])
+                                  HiveProject(brand_id=[$0], class_id=[$1], category_id=[$2], $f3=[$3])
```

Comment:
I am not sure why these names changed (it happened in a few other q files too). Since I have to regenerate a few more q files, I will explore this and comment back.